### DISCUSSION TITLE
How do I get rid of this annoying pop-up every time I accept a suggestion from copilot?

### DISCUSSION THREAD

**Discussion by m-j-biggs** [zed-industries/zed#11976] [Feature Requests]
I have just started to use Github copilot and it looks great ... except everytime I accept a suggestion the following dialogue box pops up annoyingly. I press the 'x' to get rid of it, but I would rather it just never comes up. Any advice? Many thx in advance.

Also, the language used in the pop-up is not English despite my language being set as English.  

![popup](https://github.com/zed-industries/zed/assets/166030238/19d99f65-5cd9-4fbf-af48-cd07ccca490a)

**Comment by versecafe** [comment]
I have no idea why it is in what appears to be Javanese? hard to tell with how blurred it is but still odd, the pop up seems to be the same one that comes when you run Zed in development mode, on platforms like MacOS there is the option to select `Always Allow` for requests to access keychain or platform equivalent

<img width="546" alt="image" src="https://github.com/zed-industries/zed/assets/147033096/e0fe4d12-7c51-4d3e-a0ef-a3d8e171fea0">

This should be solvable by just entering your password in and the pop up should go away, but I could be wrong about the menu cause, if so sorry for the inconvenience.

**Reply by d1y** [reply]
The same question, I want to know how to prevent the pop-up window from popping up in the future. I already trust it

**Reply by versecafe** [reply]
If you're rebuilding the app not much you can do since your OS will treat it as a new application each time, you should be able to avoid this by signing the app, and as long as the signature is consistent it shouldn't need to ask again, in theory at least for macOS and windows, for Linux I'm not sure.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
